src/internal/m365/onedrive/collection.go

```diff
@@ -515,7 +515,14 @@ func (oc *Collection) populateDriveItem(
 	// Fetch metadata for the file
 	itemMeta, itemMetaSize, err = downloadItemMeta(ctx, oc.handler, oc.driveID, item)
 	if err != nil {
-		errs.AddRecoverable(ctx, clues.Wrap(err, "getting item metadata").Label(fault.LabelForceNoBackupCreation))
+		// Skip deleted items
+		if clues.HasLabel(err, graph.LabelStatus(http.StatusNotFound)) || graph.IsErrDeletedInFlight(err) {
+			logger.CtxErr(ctx, err).With("skipped_reason", fault.SkipNotFound).Info("item not found")
+			errs.AddSkip(ctx, fault.FileSkip(fault.SkipNotFound, oc.driveID, itemID, itemName, graph.ItemInfo(item)))
+		} else {
+			errs.AddRecoverable(ctx, clues.Wrap(err, "getting item metadata").Label(fault.LabelForceNoBackupCreation))
+		}
```

Comment:
Don't add skips for deleted items, please.  Skips should be reserved for non-transient cases where we deliberately ignore an existing item.  Race conditions with item deletions are transient, and does not pass over an existing item.

```suggestion
		if !clues.HasLabel(err, graph.LabelStatus(http.StatusNotFound)) && !graph.IsErrDeletedInFlight(err) {
			errs.AddRecoverable(ctx, clues.Wrap(err, "getting item metadata").Label(fault.LabelForceNoBackupCreation))
		}
```